### ISSUE TITLE
Allow multiple remotes

### ISSUE THREAD

**Issue by vizo** [dingjie/atom-sync#10]
It would be nice to have multiple remotes for one project.
I prefer to use one for server and another for backup.

**Comment by hultberg** [comment]
:+1:

**Comment by nickv2002** [comment]
Yep. Some of work requires code in the same project running on multiple servers.  Hard to use this without multiple remote support.

**Comment by wxiang7** [comment]
+1
